Sources/SwiftCliCore/Chartify.swift

```diff
@@ -11,12 +11,12 @@ public final class Chartify {
         let input = "g1"
         let isValid = InputValidator().validate(pattern: input)
         if (isValid == true){
-            let patternArray = InputValidator().ArrayMaker(cleanedpattern: input)
+            let patternArray = InputValidator().arrayMaker(cleanedpattern: input)
             print(ChartConstructor().make_chart(stitch_array: patternArray))
         }
         else{
             print("Your input was not formatted correctly. Please include only allowed stitches seperated by \\n for line breaks. Current allowed input includes:")
-            for s in allowed_stitches{
+            for s in allowedStitches{
                 print(s)
```

Comment:
I think this will print all the allowed stitches without any delimiters in between (i.e. `"k1p1sskk2togyom1\n"`).

You might actually be able to do something like:
```{swift}
allowedStitchesString = allowedStitches.joined(separator: ", ")
print("Your input was not formatted correctly. Please include only allowed stitches seperated by \\n for line breaks. Current allowed input includes: \(allowedStitchesString)")
```